blog/electron-21-0.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Electron 21.0.0
+date: 2022-09-27T00:00:00.000Z
+authors:
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+    - name: vertedinde
+      url: 'https://github.com/vertedinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+    - name: georgexu99
+      url: 'https://github.com/georgexu99'
+      image_url: 'https://github.com/georgexu99.png?size=96'
+slug: electron-21-0
+
+---
+
+Electron 20.0.0 has been released! It includes upgrades to Chromium `106`, V8 `10.4`, and Node.js `16.16.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 21.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
```

Comment:
> download it from our [releases website](https://www.electronjs.org/releases/stable)

I've never really paid attention to this paragraph, but I don't think the releases website has a download link on it. It _does_ link to the GitHub release, though. Do we want to amend that?

Reply:
Oof, good catch, I think that's been there for a long time. I'll relink to the GH releases page 👍

Reply:
Actually, just took a look at our releases website page, and it's still listing the betas/alphas as 20-x-y. 😬 Will see if that can get updated

Reply:
@VerteDinde this would require a manual bump in our old `electron-releases` package and a manual redeploy in Heroku.

We should link to https://releases.electronjs.org/releases/stable instead.